spec/exporters/patients_with_history_exporter_spec.rb

```diff
@@ -379,4 +379,40 @@
       end
     end
   end
+
+  describe "#csv_enumerator" do
+    it "enumerates the rows of patient records" do
+      Timecop.freeze do
+        timestamp = ["Report generated at:", Time.current]
+        enumerator = subject.csv_enumerator(Patient.all)
+
+        expect(enumerator.next).to eq([timestamp, measurement_headers, headers])
+        expect(enumerator.next).to eq([fields])
+      end
+    end
+
+    context "when batch size is provided" do
+      let!(:patient2) {
```

Comment:
We can get rid of this let! by creating the patient either in the test block directly.